src/Components/CategoryCard/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState} from 'react';
+import './style.component.css'
+import Dropdown from './dropdown'
+
+
+function CategoryCard({title, imageUrl}) {
+    const [active, setActive] = useState(false);
+
+
+    function toggle(){
+        setActive(!active)
+
+    }
+
+    return (
+        <div className={`${active ? "category-card" : "category-card-white"}`} >
```

Comment:
Evitar atribuir nomes de classes que identificam quais propriedades ele atribui, como "white" por exemplo, uma vez que caso queiramos mudar essa propriedade nós teriamos que mudar o nome da classe, o que não é legal

Talvez uma boa mudança seja:

```suggestion
        <div className={`${active ? "category-card" : "category-card-active"}`} >
```